src/NextButton.js

```diff
@@ -0,0 +1,38 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import NextLink from '@moxy/next-link';
+
+export const NextButton = forwardRef(({ children, onClick, href, className, ...rest }, ref) => {
```

Comment:
I think we can dismiss `onClick` here since it will could be passed as `rest` :+1: